pkg/scripts/environment_test.go

```diff
@@ -43,30 +41,30 @@ func equalArrayContents(a []string, b []string) bool {
 	return true
 }
 
-func TestConvertEnvironmentToDocker(t *testing.T) {
-	testValues := []string{
-		"Value1",
-		"$Value1",
-		"${Value1}",
-		"`Value1`",
-		`"`,
-	}
-	var charValues []string
-	for ch := rune(32); ch < 127; ch++ {
-		charValues = append(charValues, "AB"+string(ch)+"CD", "AB\\"+string(ch)+"CD")
-	}
-	for _, testValue := range append(testValues, charValues...) {
-		list := api.EnvironmentList{{Name: "TEST", Value: testValue}}
-		converted := ConvertEnvironmentToDocker(list)
-		config, err := dockerfile.BuildFromConfig(&container.Config{}, []string{converted})
-		if err != nil {
-			t.Fatalf("Unexpected error building using Dockerfile contents %v: %v", converted, err)
-		}
-		if len(config.Env) != 1 {
-			t.Errorf("Unexpected result. Expected 1 environment variable, got config %#v", config)
-		}
-		if config.Env[0] != "TEST="+testValue {
-			t.Errorf("Unexpected result. Expected: %s=%#v -> %q -> %s=%v. Got: %#v", list[0].Name, testValue, converted, list[0].Name, testValue, config.Env[0])
-		}
-	}
-}
+// func TestConvertEnvironmentToDocker(t *testing.T) {
+// 	testValues := []string{
+// 		"Value1",
+// 		"$Value1",
+// 		"${Value1}",
+// 		"`Value1`",
+// 		`"`,
+// 	}
+// 	var charValues []string
+// 	for ch := rune(32); ch < 127; ch++ {
+// 		charValues = append(charValues, "AB"+string(ch)+"CD", "AB\\"+string(ch)+"CD")
+// 	}
+// 	for _, testValue := range append(testValues, charValues...) {
+// 		list := api.EnvironmentList{{Name: "TEST", Value: testValue}}
+// 		converted := ConvertEnvironmentToDocker(list)
```

Comment:
Seems like you could do some validation of the return from `ConvertEnvironmentToDocker` without calling dockerfile functions

Reply:
Yeah - the actual function is pretty trivial. I'll add a true unit test.

Reply:
@gabemontero restored coverage via a simple unit test. PTAL

Reply:
cool